spec/factories/users.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :user do
+    email 'email@email.com'
+    name 'nisse'
```

Comment:
I really would prefer if you guys use capital letters in names. `nisse` would probably prefer to be called `Nisse`